src/index.js

```diff
@@ -1,9 +1,12 @@
 const path = require('path')
 
+const { satisfies } = require('semver')
```

Comment:
I don't really want to add another dependency if possible, do we really need the 'full' `semver` package or would a smaller utility function also solve it e.g 

```js
const SEMVER_RE = /\^?\~?(\d+)\.(\d+).(\d+)/

function semver (version) {
    const [ _, major, minor, patch ] = version.match(SEMVER_RE)
    
    return {
      major,
      minor,
      patch
   }
}

const version = semver(postcssPkg.version)

// ...
```

Please see the comment below for further discussion

Reply:
parsing semver with a simple regex is a generally a bad idea, and can fail in unexpected ways. I get not wanting another dependency but css-loader already does this and it's probably more worthwhile to match the behavior esp since the dependency is most likely already installed

Reply:
You regexp is invalid, parsing version using regexp is a bad idea

Reply:
Why? I mean I simply not convinced this requires to have all blows and whistles of the semver spec to work and therefore the whole `semver` as a dependecy might be avoidable, but let's not hung ourselves up on this for to long, like mentioned above I will open an issue in the `postcss` repo